### DISCUSSION TITLE
data是空数组的时候可以提供一个空坐标样式吗，现在是一片空白

### DISCUSSION THREAD

**Discussion by KuduroJS** [antvis/G2#5658] [Q&A] [answered]
当数组为空的时候，是一片空白，给人的感觉比较困惑。
![image](https://github.com/antvis/G2/assets/101621239/20fa2147-8053-4f17-906a-e11591cc11a0)

希望即使data空的时候，坐标系等依然按配置显示。

类似这种：
![image](https://github.com/antvis/G2/assets/101621239/30cba7db-72bc-47fa-bff7-b79fcf537521)

有相关实现方法吗？

空白demo：
```typescript
/**
 * A recreation of this demo: https://observablehq.com/@d3/bar-chart
 */
import { Chart } from '@antv/g2';

const chart = new Chart({
  container: 'container',
  autoFit: true,
});

chart
  .interval()
  .data([])
  .encode('x', 'letter')
  .encode('y', 'frequency')
  .axis('y', { labelFormatter: '.0%' });

chart.render();
```

**Comment by pearmini** [comment, marked as answer]
理论上讲不同的图表在空数据的情况下展现是不同的，G2 没有办法提供通用解决方案。可以在 data 为空的时候，自己 mock 数据，隐藏 mark 来实现如上效果。

**Reply by KuduroJS** [reply]
了解，谢谢
